man/systemd.exec.xml

```diff
@@ -1311,6 +1311,19 @@
         <citerefentry><refentrytitle>tmpfiles.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DenyWriteExecuteMemory=</varname></term>
+
+        <listitem><para>Takes a boolean argument. If set, a seccomp filter is added to reject
+        <citerefentry><refentrytitle>mmap</refentrytitle><manvolnum>2</manvolnum></citerefentry>
+        system calls with both <constant>PROT_EXEC</constant> and <constant>PROT_WRITE</constant> set
+        and <citerefentry><refentrytitle>mprotect</refentrytitle><manvolnum>2</manvolnum></citerefentry>
+        system calls with <constant>PROT_EXEC</constant> set. This option may be incompatible with dynamic
+        code generators, programs using trampolines and some Xorg drivers.
```

Comment:
Also, I wonder if this shouldn't better be called "MemoryDenyWriteExecute=" or so. This would match MemoryLimit=, i.e. put the subsystem something has an effect on first.